App/components/buttons/Button.tsx

```diff
@@ -1,14 +1,14 @@
 import React from 'react'
 import { Text, TouchableOpacity } from 'react-native'
 
-import { Buttons, heavyOpacity } from '../../theme'
+import { useThemeContext } from '../../utils/themeContext'
 
 export enum ButtonType {
   Primary,
   Secondary,
 }
 
-interface ButtonProps {
+export interface ButtonProps {
```

Comment:
Not for this PR, it will be needed eventually for the bifold as framework, but I guess we also need to have a conversation about how to implement models/interfaces that downstream projects can use. I will remove the export for now.